src/index.ts

```diff
@@ -0,0 +1,219 @@
+//import gameCard from './game-cards';
+//import IGameCard from './game-cards/game-card.interface';
+import workWithStorage from './utilites';
+import gameWithFilters from './utilites/games_with_filters';
+import { sliderOne, sliderTwo, fillColor } from './utilites/slider';
+import { IFilters } from './utilites';
+import { showCart, clearCart } from './utilites/cart';
+import './style/index.css';
+
+
+const gameCard = new gameWithFilters();
+const filters = new workWithStorage();
+const state: IFilters = filters.get();
+fillColor();
+gameCard.filters(state);
+
+const slider1 = document.querySelector('#slider-1') as HTMLInputElement;
+slider1.addEventListener('input', sliderOne);
+
+const slider2 = document.querySelector('#slider-2') as HTMLInputElement;
+slider2.addEventListener('input', sliderTwo);
+
+const searchField = document.querySelector('.filters__search') as HTMLInputElement;
+searchField.value = state.search;
+searchField.addEventListener('input', (e) => {
+    filters.setSearch((e.target as HTMLInputElement).value);
+    gameCard.filters(filters.get());
+});
+
+const selectSearch = document.querySelector('.filters__sort') as HTMLInputElement;
+selectSearch.value = state.sortBy;
+selectSearch.addEventListener('input', (e) => {
+    filters.sortBy((e.target as HTMLInputElement).value);
+    gameCard.filters(filters.get());
+});
+
+//adding visual distinction for active buttons
+
+function activeClass(btn: Element | null) {
+    if (!btn){
+        return;
+    }
+    btn.classList.toggle('active');
+}
+function checkButton(state: IFilters){
+    state.platform.forEach(el => {
+        if (el === 'PS'){
```

Comment:
You should create separate interfaces/enum instead of hardcoded strings (as 'PS', 'Valve' etc.). In that case you can change name only in one place, not in all places where you use this name.